fixture/vcr_cassettes/auth0-invalid-code.json

```diff
@@ -0,0 +1,47 @@
+[
```

Comment:
The `fixture/` files are already present in `test/fixtures/` so they shouldn't be in this pull request. There's maybe a path issue in your setup that duplicated the files here.

Reply:
You're right. Maybe I run some command that I shouldn't have done on Terminal.

I removed the `fixture/` folder and pushed the changes :).